google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -460,9 +460,14 @@ def deploy_to_local_endpoint(
 
         Args:
             artifact_uri (str):
-                Optional. The Cloud Storage path to the directory containing the Model artifact
-                and any of its supporting files. The AIP_STORAGE_URI environment variable will
-                be set to this uri if given; otherwise, an empty string.
+                Optional. The path to the directory containing the Model artifact and any of its
+                supporting files. The path is either a GCS uri or the path to a local directory.
+                If this parameter is set to a GCS uri, you may need to specify `credential_path`.
+                If this parameter is set to a path to a local directory, the directory will be
+                mounted to the default temporary directory in the container.
+                The AIP_STORAGE_URI environment variable will be set to this parameter if it's a
```

Comment:
I don't think we need to mention `AIP_STORAGE_URI` here. I don't think CPR users should need to know about AIP_STORAGE_URI. Maybe something like...

If this parameter is set to a GCS uri:
(1) `credential_path` must be specified for local prediction.
(2) The GCS uri will be passed directly to `Predictor.load`.
If this parameter is a local directory:
(1) The directory will be mounted to a default temporary model path.
(2) The mounted path will be passed to `Predictor.load`.
If this is a GCS path, it will be passed directly to Predictor.load. If it's a local path,